src/components/core/geoChart/geoChartTooltip.ts

```diff
@@ -0,0 +1,85 @@
+// (C) 2020 GoodData Corporation
+import get = require("lodash/get");
+import isEmpty = require("lodash/isEmpty");
+import mapboxgl from "mapbox-gl";
+import { DEFAULT_PUSHPIN_COLOR_VALUE } from "../../../constants/geoChart";
+
+function isTooltipItemValid(item: { title?: string; value?: string } = {}): boolean {
```

Comment:
please make type for `{ title?: string; value?: string }`  and reuse in function `getTooltipItemHtml`

Reply:
Fixed